test/services/cloudsearchdomain.spec.coffee

```diff
@@ -49,8 +49,16 @@ describe 'AWS.CloudSearchDomain', ->
       params = { query: 'foo' }
       req = build('search', params)
       expect(req.headers).not.to.have.property('Authorization')
-    
+
     it 'signs request if credentials are provided', ->
       params = { query: 'foo' }
       req = build('search', params)
       expect(req.headers).to.have.property('Authorization')
+
+    it 'converts the GET request to POST for search operation', ->
```

Comment:
I will add a test for `suggest`, as this is the only operation that uses a `GET`.